styles/scss/mixins/_underline-text.scss

```diff
@@ -0,0 +1,109 @@
+///  This mixin has 2 possible way of using it.
+///  First is to include in the class you want to hover over while text
+///  you want to underline needs to have attached class .underline-text
+///  or you can send your custom class name via parameter $element.
+///  Also you have to set $wrapper parameter to true.
+///  Second is to just include it in whichever element you want. In this
+///  case, parameter $wrapper should stay unchanged.
+///  Other parameters are optional and they allow you customize your transition.
```

Comment:
First of all you can add some empty lines in the description (tho I think it renders the same in the sassdoc itself).

```
///  This mixin has 2 possible way of using it.
///
///  First is to include in the class you want to hover over while text
///  you want to underline needs to have attached class .underline-text
///  or you can send your custom class name via parameter $element.
///
///  Also you have to set $wrapper parameter to true.
///
///  Second is to just include it in whichever element you want. In this
///  case, parameter $wrapper should stay unchanged.
///
///  Other parameters are optional and they allow you customize your transition.
```

Second, the description is markdown so you can highlight things.

```markdown
///  This mixin has 2 possible ways of using it.
///
/// #### Example 1
///
///  Include in the class you want to hover over while text
///  you want to underline needs to have attached class `.underline-text`
///  or you can send your custom class name via parameter `$element`.
///
///  Also you have to set `$wrapper` parameter to true.
///
/// #### Example 2
///  
///  Just include it in whichever element you want. In this
///  case, parameter `$wrapper` should stay unchanged.
///
///  Other parameters are optional and they allow you customize your transition.
```

> Include in the class you want to hover over while text you want to underline needs to have attached class `.underline-text`

The first way's explanation is kinda confusing. I need to include the mixing AND add the class of `underline-text` to the element? 

> or you can send your custom class name via parameter `$element`.

So instead of the both things above, I can just add the class as parameter?

Other than that this is the best thing ever ❤️

Reply:
I agree the first example is confusing to me. The text is in a separate class that is wrapped with the class that holds the hover?

Add a comma after `Also`, and `the` before `$wrapper` like

 ```
Also, you have to set the `$wrapper` parameter to true.
```